src/main/cocoa/CocoaWindow.mm

```diff
@@ -0,0 +1,731 @@
+/*
+ * Copyright (C) 2025 Linux Studio Plugins Project <https://lsp-plug.in/>
+ *           (C) 2025 Vladimir Sadovnikov <sadko4u@gmail.com>
+ *
+ * This file is part of lsp-ws-lib
+ * Created on: 1 июл. 2022 г.
+ *
+ * lsp-ws-lib is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * lsp-ws-lib is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with lsp-ws-lib. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <lsp-plug.in/common/types.h>
+
+#ifdef PLATFORM_MACOSX
+
+#import <Cocoa/Cocoa.h>
+#include <iostream>
```

Comment:
Do we really need <iostream> here?
You can use `lsp_trace`, `lsp_info` and other functions from `<lsp-plug.in/common/debug.h>` for printing.